### PR TITLE
PROB-2187 [Zigbee Dimmer/Switch Power] Fix loss of manual control over GE/Jasco switches

### PR DESCRIPTION
**Pull request by tpmanley**
https://smartthings.atlassian.net/browse/PROB-2187

### CODE REVIEW

File: devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy [hunk 1]
```diff
@@ -79,7 +79,12 @@ def on() {
 
 def refresh() {
     Integer reportIntervalMinutes = 5
-    zigbee.onOffRefresh() + zigbee.simpleMeteringPowerRefresh() + zigbee.electricMeasurementPowerRefresh() + zigbee.onOffConfig(0,reportIntervalMinutes * 60) + zigbee.simpleMeteringPowerConfig() + zigbee.electricMeasurementPowerConfig()
+    def cmds = zigbee.onOffRefresh() + zigbee.simpleMeteringPowerRefresh() + zigbee.electricMeasurementPowerRefresh() + zigbee.onOffConfig(0, reportIntervalMinutes * 60) + zigbee.simpleMeteringPowerConfig() + zigbee.electricMeasurementPowerConfig()
+    if (device.getDataValue("manufacturer") == "Jasco Products") {
+        // Some versions of hub firmware will incorrectly remove this binding causing manual control of switch to stop working
+        cmds << ["zdo bind 0x${device.deviceNetworkId} 2 1 0x0006 {${device.zigbeeId}} {${device.zigbeeId}}"]
```
Comment (varzac) [inline devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy]: We are double binding the on/off cluster on configure, but only doing it once for level.  I don't think we need this in 2 places.

**Review by varzac — DISMISSED** [review]

File: devicetypes/smartthings/zigbee-dimmer-power.src/zigbee-dimmer-power.groovy [hunk 2]
```diff
@@ -130,6 +130,11 @@ def configure() {
         }
         // Level Control Cluster, command Move to Level, level start level, transition time 0
         cmds << zigbee.command(zigbee.LEVEL_CONTROL_CLUSTER, 0x00, sprintf("%02X0000", startLevel))
+    } else if (device.getDataValue("manufacturer") == "Jasco Products") {
+        // Some versions of hub firmware will incorrectly remove this binding causing manual control of switch to stop working
+        // These needs to be the first binding table entries because the device will automatically write these entries each time it restarts
+        cmds += ["zdo bind 0x${device.deviceNetworkId} 2 1 0x0006 {${device.zigbeeId}} {${device.zigbeeId}}", "delay 2000",
```
Comment (greens) [inline devicetypes/smartthings/zigbee-dimmer-power.src/zigbee-dimmer-power.groovy]: is this different than

`zigbee.addBinding(___)`?

also tsk tsk for magic numbers

Comment (tpmanley) [reply devicetypes/smartthings/zigbee-dimmer-power.src/zigbee-dimmer-power.groovy]: `zigbee.addBinding` doesn't support changing the destination EUID

**Review by varzac — DISMISSED** [review]

**Review by greens — DISMISSED** [review]

**Comment by tpmanley** [comment]
Will aim to merge once QA has finished testing

**Review by rappleg — APPROVED** [review]

**Review by varzac — APPROVED** [review]

**Pull request merged**